src/storage/v2/replication/serialization.hpp

```diff
@@ -33,7 +33,7 @@ class Encoder final : public durability::BaseEncoder {
 
   void WriteDouble(double value) override;
 
-  void WriteString(const std::string_view &value) override;
+  void WriteString(const std::string_view value) override;
```

Comment:
```suggestion
  void WriteString(std::string_view value) override;
```

Reply:
Uh, my vim grep didn't go the way I wanted. Nice catches!

Reply:
So you love Javascript? Maybe I should endorse you :eye: